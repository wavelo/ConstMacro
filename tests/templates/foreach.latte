
{default $data = [['title' => 'title #0'], ['title' => 'title #1'], ['title' => 'title #2']]}

<div>
	{foreach $data as [title:]}
	<p>{$title}</p>
	{/foreach}
</div>

<div>
	{foreach $data as $key => [title:]}
	<p>{$title}</p>
		{foreach $data as $key => [title:]}
		<p>{$title}</p>
		{/foreach}
	<p>{$title}</p>
	{/foreach}
</div>

<div n:inner-foreach="$data as $key => [title:], $props">
	<p>{$title}</p>
	<p>{$props['title']}</p>
</div>

<div>
	{foreach $data as $key => $val}
	<p>{$val[title]}</p>
	{/foreach}
</div>
