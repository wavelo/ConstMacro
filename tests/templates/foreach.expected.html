
<div>
	<p>title #0</p>
	<p>title #1</p>
	<p>title #2</p>
</div>

<div>
	<p>title #0</p>
		<p>title #0</p>
		<p>title #1</p>
		<p>title #2</p>
	<p>title #0</p>
	<p>title #1</p>
		<p>title #0</p>
		<p>title #1</p>
		<p>title #2</p>
	<p>title #1</p>
	<p>title #2</p>
		<p>title #0</p>
		<p>title #1</p>
		<p>title #2</p>
	<p>title #2</p>
</div>

<div>
	<p>title #0</p>
	<p>title #0</p>
	<p>title #1</p>
	<p>title #1</p>
	<p>title #2</p>
	<p>title #2</p>
</div>

<div>
	<p>title #0</p>
	<p>title #1</p>
	<p>title #2</p>
</div>
